src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -977,6 +978,12 @@ private function registerSerializerConfiguration(array $config, ContainerBuilder
             $definition->addTag('serializer.normalizer', array('priority' => -900));
         }
 
+        if (class_exists(YamlEncoder::class)) {
```

Comment:
You must also ensure ``symfony/yaml`` is loaded as it is [a dev dependency of the ``framework-bundle``](https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/composer.json#L52).